core/src/test/scala/matryoshka/spec.scala

```diff
@@ -466,9 +474,11 @@ class FixplateSpecs extends Specification with ScalaCheck with ScalazMatchers {
       }
     }
 
-    def extractFactors: Coalgebra[Exp, Int] = x =>
+    // TODO: Why do we need to explicitly call the implicit here? Is it because
```

Comment:
I _think_ I did. Ideally users should be able to use these types or functions interchangeably. I’m not sure if it worked when passed directly to a fold, but I recall scala complaining that `generalize` wasn’t a member of `Int => Exp[Int]`, when trying to convert it to a generalized fold.